pom.xml

```diff
@@ -92,6 +92,7 @@
 		<module>struts2-jquery-grid-showcase</module>
 		<module>struts2-jquery-mobile-showcase</module>
 		<module>struts2-jquery-archetypes</module>
+		<module>struts2-jquery-plugin-tests</module>
```

Comment:
What do you think about naming this module `struts2-jquery-integration-tests` or something like that?